### DISCUSSION TITLE
Alternative authentication using a GCP Project

### DISCUSSION THREAD

**Discussion by robcerda** [mxrch/GHunt#328] [Ideas]
Hi there,

I managed to obtain the gaia_id using a Google Cloud Project for authentication, as opposed to the cookies outlined in your instructions.

I'm curious if anyone would like to work together to see what else can be retrieved using a Google Cloud Project along with Google's apis.

**Comment by mxrch** [comment]
> Hi there,
> 
> I managed to obtain the gaia_id using a Google Cloud Project for authentication, as opposed to the cookies outlined in your instructions.
> 
> I'm curious if anyone would like to work together to see what else can be retrieved using a Google Cloud Project along with Google's apis.
> 

Hello, so you used a service account for the login ?

I'm working on it too, you can join our Discord Server and talk there ! https://discord.gg/4RTHGMdWQx 😊
